doc/development-context-aware-applications/introduction.md

```diff
@@ -4,9 +4,7 @@ consume context information at large scale and exploit it to transform
 your application into a truly smart application.  
 
 Context information is represented through values assigned to attributes
-that characterize those entities relevant to your application. [The
-Context Broker GE (open source reference implementation:
-Orion)](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
+that characterize those entities relevant to your application. [The Context Broker GE (open source reference implementation: Orion)](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
```

Comment:
We deleted new lines characters inside the link. We can undo this change if you want